app/controllers/users_controller.rb

```diff
@@ -158,8 +159,24 @@ def standing
 
 private
 
+  # If viewing user at "/u/tEsT" but username is actually "test", then redirect to "/u/test".
+  def redirect_case_mismatch
+    target = User.where(username: params[:username]).first
```

Comment:
This and `@showing_user = User.where(:username => params[:username]).first` up on line 10 are identical, so we're duplicating the query. Can you update the PR to avoid querying the same record twice?